### DISCUSSION TITLE
Welcome to Q&A

### DISCUSSION THREAD

**Discussion by greyhatguy007** [greyhatguy007/Machine-Learning-Specialization-Coursera#42] [Q&A]
We’re using Q&A as a place to connect with other members of our community to learn and grow. We hope that you:
  * Ask questions related to the topic.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. 
  
**Note** : Kindly refrain from asking answers bluntly instead of trying it yourself first. You can ask questions like how it works, why it works, how to optimize.

*Share your knowledge and happy learning :)*

**Comment by EasperOne** [comment]
Hey, big thanks to you!

**Reply by JingRoboLab** [reply]
which python and TF version does this repo use?

**Reply by HussainDitta** [reply]
For python i used 3.12 not sure which version way yeah for TF
